opensrp-reveal/src/main/java/org/smartregister/reveal/adapter/ChildRegisterAdapter.java

```diff
@@ -0,0 +1,56 @@
+package org.smartregister.reveal.adapter;
+
+import android.support.annotation.NonNull;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import org.smartregister.reveal.R;
+import org.smartregister.reveal.model.Child;
+import org.smartregister.view.ListContract;
+import org.smartregister.view.adapter.ListableAdapter;
+import org.smartregister.view.viewholder.ListableViewHolder;
+
+import java.util.List;
+
+public class ChildRegisterAdapter extends ListableAdapter<Child, ListableViewHolder<Child>> {
+
+    public ChildRegisterAdapter(List<Child> items, ListContract.View<Child> view) {
+        super(items, view);
+    }
+
+    @NonNull
+    @Override
+    public ListableViewHolder<Child> onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
+        View view = LayoutInflater.from(parent.getContext()).inflate(R.layout.child_register_list_row, parent, false);
+        return new ChildViewHolder(view);
+    }
+
+    public static class ChildViewHolder extends ListableViewHolder<Child> {
+
+        private TextView tvName, tvDetails;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=5414138)